src/components/Banner.tsx

```diff
@@ -43,7 +43,7 @@ const Banner: React.FC<BannerProps> = ({
     useEffect(() => {
         if (data) {
             const rand = Math.floor((Math.random() * data.length) / 2);
-            const path = `https://image.tmdb.org/t/p/original${data?.[rand]?.banner_path}` || null;
+            const path = `https://image.tmdb.org/t/p/original${data?.[rand]?.banner_path}`;
```

Comment:
I think this was incorrect before, and still won't properly catch malformed data. 
```suggestion
            const path = data?.[rand]?.banner_path ? `https://image.tmdb.org/t/p/original${data?.[rand]?.banner_path}` : null;
```